docs/4.4. Behaviour - Access Tokens.md

```diff
@@ -132,12 +132,16 @@ The `value` corresponding to each permission key is a JSON array containing URL
 permitted to perform the specific request against. The asterisk wildcard ('`*`') is permitted in the path
 specifiers.
 
-A '`*`' wildcard can be used to replace zero or more path characters, and may encompass multiple path segments (e.g.
-`/single/*` and `/single/senders/*/constraints` would both correspond to
-`/single/senders/ea388089-9ffb-4a81-b109-a19da845b3b6/constraints`). It is RECOMMENDED that wildcards are used in
-place of NMOS resource identifiers, such as UUIDs, to minimise token size. Specific identifiers MAY
+A '`*`' wildcard can be used to replace zero or more characters (this is equivalent to the `.*` regex pattern). The 
+wildcard may encompass multiple path segments, may be used multiple times, and may appear at any position in the path 
+permission (e.g. `single*` or `single/senders/*/constraints` would both match with
+`/x-nmos/connection/v1.1/single/senders/ea388089-9ffb-4a81-b109-a19da845b3b6/constraints`). It is RECOMMENDED that the
+wildcard is used in place of NMOS resource identifiers, such as UUIDs, to minimise token size. Specific identifiers MAY
 be used in path specifiers if defining access to a small number of resources (such as a single node or device.)
 
+Resource Servers SHALL apply [URL Normalization][URL Normalization] in order to prevent double dot URL `..` introducing 
```

Comment:
```suggestion
Resource Servers SHALL apply [URL Normalization][URL Normalization] in order to prevent double dot URL segments (i.e. `..`) due to them introducing 
```

Reply:
l like adding 'segments', but dislike 'due to them', since URL normalization doesn't prevent the `..` segments themselves, it prevents them introducing the vulnerability.

```suggestion
Resource Servers SHALL apply [URL Normalization][URL Normalization] in order to prevent double dot path segments, '`..`', introducing 
```